### PR TITLE
More author[] -> authors[] 1.60 fixes

### PR DESCRIPTION
**Pull request by Killswitch00**

### CODE REVIEW

File: optionals/cache_disable/config.cpp [hunk 1]
```diff
@@ -7,7 +7,8 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"CBA_Extended_EventHandlers", "CBA_Main"};
         version = VERSION;
-        author[] = {"Sickboy"};
+        author = "$STR_CBA_Author";
```
Comment (jonpas) [inline optionals/cache_disable/config.cpp]: `CSTRING(Author)`?

Comment (Killswitch00) [inline optionals/cache_disable/config.cpp]: That would become `$STR_CBA_cache_disable_Author`, wouldn't it?

Comment (jonpas) [inline optionals/cache_disable/config.cpp]: Right, that is true. My bad, I am used to having it in one PBO and use `ECSTIRNG(pbo,Author)`.

File: optionals/jr_disable_long_scopes_on_short_mg_rail/config.cpp [hunk 2]
```diff
@@ -9,7 +9,8 @@ class CfgPatches
 		requiredVersion = REQUIRED_VERSION;
 		requiredAddons[] = {"cba_jr"};
 		version = VERSION;
-		author[] = {"Robalo"};
+		author = "$STR_CBA_Author";
+		authors[] = {"Robalo"};
         authorUrl = "https://github.com/CBATeam/CBA_A3";
```
Comment (commy2) [inline optionals/jr_disable_long_scopes_on_short_mg_rail/config.cpp]: We should convert all tabs to 4 spaces just to avoid this ^

Comment (Killswitch00) [inline optionals/jr_disable_long_scopes_on_short_mg_rail/config.cpp]: Fixed

**Pull request merged**